site/content/docs/5.3/utilities/link.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: docs
+title: Link
+description: Link utilities are used to stylize your anchors to adjust their color, opacity, underline offset, underline color, and more.
+group: utilities
+toc: true
+added: 5.3
+---
+
+## Link opacity
+
+Change the alpha opacity of the link `rgba()` color value with utilities. Please be aware that changes to a color's opacity can result in contrast ratios below the WCAG minimum.
```

Comment:
suggest rewording "in contrast ratios below the WCAG minimum" to something more like

`can lead to links with [*insufficient* contrast]({{< docsref "getting-started/accessibility#color-contrast" >}}).`